README.md

```diff
@@ -31,9 +31,11 @@ A complete toolkit for the NEO blockchain, including:
 
 ## Installation
 
+Go: 1.12+
```

Comment:
Not really, we even have 1.11 configured in CircleCI. Also, it has no relation to docker again.